Server/ReasnAPI/ReasnAPI/Services/ParameterService.cs

```diff
@@ -71,6 +71,25 @@ public void DeleteParameter(int parameterId)
         context.Parameters.Remove(parameter);
         context.SaveChanges();
     }
+
+    public void ForceDeleteParameter(int parameterId)
```

Comment:
Should also think about that we don't have parameterId in DTO, correct me if I'm wrong

Reply:
chenged the logic: we pass the dto and then is is checking by key and value